packages/ag-charts-website/vitest.config.mjs

```diff
@@ -8,11 +7,8 @@ function resolvePath(srcPath) {
 
 export default getViteConfig({
     root: __dirname,
-    cacheDir: '../../node_modules/.vite/test',
```

Comment:
Is the caching disabled now, or is the path defaulted somewhere else now?

Reply:
This defaults to `node_modules/.vite`: https://vite.dev/config/shared-options.html#cachedir

My thinking was that it was in the `.vite` directory already. Does it need to specifically be in `.vite/test` for nx caching or anything?